src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const rentPrice = 40;
+
+  if (days >= shortTerm && days < longTerm) {
```

Comment:
The condition `days >= shortTerm && days < longTerm` correctly identifies the short-term rental period. Ensure that the discount logic aligns with the task requirements.